src/main/java/com/google/api/generator/gapic/composer/SampleCodeHelperComposer.java

```diff
@@ -50,13 +55,30 @@ public static TryCatchStatement composeRpcMethodSampleCode(
 
   private static TryCatchStatement composeUnaryRpcMethodSampleCode(
       Method method, List<MethodArgument> arguments, TypeNode clientType) {
-    // TODO(summerji): compose sample code for unary rpc method.
+    // Assign each method arguments with default value.
```

Comment:
Can we please add TODOs for adding unit tests for each private method here?

Reply:
Sure. Added.